fix(data tiering): rename and move (#1707)
timescale/docs: timescaledb/how-to-guides/schema-management/manage-storage.md

---
title: Manage storage using tablespaces
excerpt: Save on data storage by moving chunks between tablespaces
keywords: [storage, tablespaces]
tags: [move, manage, chunks]
---

# Manage storage using tablespaces

> You can save on storage by moving chunks between tablespaces when you run
> TimescaleDB on your own hardware. By moving older chunks to cheaper, slower

Reviewer: ```suggestion
If you are running TimescaleDB on your own hardware, you can save storage 
by moving chunks between tablespaces. By moving older chunks to cheaper, slower
```
